libraries/AP_AHRS/AP_AHRS_NavEKF.h

```diff
@@ -296,12 +299,12 @@ class AP_AHRS_NavEKF : public AP_AHRS_DCM {
 
 private:
     enum class EKFType {
-        NONE = 0,
+        NONE = 0
```

Comment:
Commas can go a the end of all these lines.   Trailing commas are just fine in enumeration lists.

Way less ugly that way and consistent with the rest of our code.

Reply:
I changed it as it didn't build in SITL